docs/developing/testing.rst

```diff
@@ -0,0 +1,41 @@
+Testing
+#######
+
+This section covers writing tests for the ``h`` codebase.
+
+Getting started
+---------------
+
+Sean Hammond has written up a `guide to getting started`_ running and writing
+our tests, which covers some of the tools we use (``tox`` and ``pytest``) and
+some of the testing techniques they provide (factories and parametrization).
+
+.. _guide to getting started: https://www.seanh.cc/posts/running-the-h-tests
```

Comment:
Feel free to steal from this if there's anything useful in there, but I wouldn't bother linking to it necessarily